eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,59 @@
+param (
+  [Parameter(mandatory = $true)]
+  $artifactName,
```

Comment:
Why do we need the artifact name? What happens if we release multiple libraries? Can we not just get the info from the path package?

Reply:
the name is per package, shouldn't be a problem if we use multiple libraries

Reply:
follow up comments below